traefik/templates/_podtemplate.tpl

```diff
@@ -560,6 +564,14 @@
           - "--providers.kubernetesingress.namespaces={{ template "providers.kubernetesIngress.namespaces" $ }}"
           {{- end }}
           {{- end }}
+          {{- with .Values.providers.file }}
+          {{- if .enabled }}
+          - "--providers.file.filename=/opt/traefik/extra-config/config.yml"
```

Comment:
we discourage the usage of `filename` related to problems with some file systems.
We recommend using `directory` instead.